app/src/main/java/com/framgia/soundclound/screen/addtracktoalbum/AddTrackViewModel.java

```diff
@@ -0,0 +1,86 @@
+package com.framgia.soundclound.screen.addtracktoalbum;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports